RecoMET/METProducers/src/GlobalHaloDataProducer.cc

```diff
@@ -14,12 +14,15 @@ using namespace reco;
 
 GlobalHaloDataProducer::GlobalHaloDataProducer(const edm::ParameterSet& iConfig)
 {
+
+  ishlt = iConfig.getUntrackedParameter< bool> ("IsHLT",false);
```

Comment:
from the implementation, it looks like this flag changes results of produced result.
If so, the parameter has to be tracked